### ISSUE TITLE
replacement for getFullPathIndex?

### ISSUE THREAD

**Issue by 123Haynes** [webjars/webjars-locator-core#29]
Hi,

I saw that you removed the method getFullPathIndex from the `WebJarAssetLocator` in https://github.com/webjars/webjars-locator-core/commit/60f01fb609844cbe7d6d39055e756bd13e35a9fb#diff-91255789c37eabd7e53b4647bc611bd0L98  

Is there a replacement for this method?
I'm currently using this method in the libsass-maven-plugin. Currently, I can pass in something like 
`"susy/sass/susy/_math.scss"`  
and get back a path like 
`META-INF/resources/webjars/susy/2.1.1/sass/susy/_math.scss`

See https://gitlab.com/haynes/libsass-maven-plugin/blob/master/src/main/java/wrm/libsass/WebJarTranslator.java for how it's currently used in the plugin.

**Comment by jamesward** [comment]
Thanks for reaching out!  Sorry about the hassle.  In your case can you use:
locator.getFullPathExact("bootstrap", "js/bootstrap.js");

For reference:

**Comment by 123Haynes** [comment]
@jamesward Thanks for your reply 😄   
I'll have to rewrite stuff a little bit and extract the name from the uri, but that works for me.  

That being said, I think 0.39 removed a bit of flexibility because now you don't have a function anymore where you can pass a custom regex matcher anymore.  
It might be worth adding that back.

**Comment by jamesward** [comment]
Yeah, I think we should be able to add more flexible filtering / params for ClassGraph (which now does the underlying classpath scanning).
